pkg/schema/spec/definitions.yaml

```diff
@@ -344,3 +344,32 @@
       isArray: false
   requiredPrivileges:
     - VEHICLE_NON_LOCATION_DATA
+- vspecName: Vehicle.AngularVelocity.Yaw
+  conversions: 
+    - originalName: yawRate
+      originalType: float64
+      isArray: false
+  requiredPrivileges:
+    - VEHICLE_NON_LOCATION_DATA
+- vspecName: Vehicle.Powertrain.ElectricMotor.CoolantTemperature
```

Comment:
is there something for the battery coolant temp? this one says ElectricMotor

Reply:
I suppose it could be:

Vehicle.Powertrain.TractionBattery.Temperature = Temperature Information for the battery pack.

Reply:
For EV's do the batteries and motor have different coolant systems?

Reply:
depends on the vehicle design, sometimes they're on the same loop, sometimes different loops, or motor is only air cooled